nemo/collections/nlp/models/intent_slot_classification/intent_slot_classification_model.py

```diff
@@ -69,9 +69,6 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
         # init superclass
         super().__init__(cfg=cfg, trainer=trainer)
 
-        # Enable setup methods.
-        IntentSlotClassificationModel._set_model_restore_state(is_being_restored=False)
```

Comment:
@vladgets is it okay to move this?

Reply:
@zeyuznv, it looks like this line was added by the TLT team. Could you comment if it's still necessary?

Reply:
@tkornuta-nvidia